zcash_client_backend/src/data_api.rs

```diff
@@ -29,6 +31,8 @@ pub mod chain;
 pub mod error;
 pub mod wallet;
 
+pub const SAPLING_SHARD_HEIGHT: u8 = sapling::NOTE_COMMITMENT_TREE_DEPTH / 2;
```

Comment:
Personally I would just have used the constant 16, but I don't feel strongly about it.